ArcSlider/ArcSlider.js

```diff
@@ -125,6 +125,14 @@ const ArcSliderBase = kind({
 		 */
 		min: PropTypes.number,
 
+		/**
+		 * Disable the knob accent color on focus.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		noFocusColor: PropTypes.bool,
```

Comment:
Why is it needed???
I think the GUI guide missed the disabled focus state.
Even when disabled, the user should be able to recognize the focus position.